### DISCUSSION TITLE
package patch request: use termux-tts-speak as an output module for speech-dispatcher

### DISCUSSION THREAD

**Discussion by sudomain** [termux/termux-packages#15783] [Ideas]
Hello,

I would've made this an issue but it wasn't quite a package request or a bug report. I recently got [`termux-tts-speak` to work as an output module for speechd](https://www.reddit.com/r/termux/comments/11l3ev0/termuxttsspeak_integration_with_speechdispatcher/). It only requires adding a line to speechd.conf and creation of a new file called termux-tts-generic.conf (details in that link).

I'm not sure how to submit such a PR, otherwise I would.

**Comment by sylirre** [comment]
Yes, you can submit a pull request. But make sure that `termux-tts` configuration is commented out by default and in build.sh added `TERMUX_PKG_SUGGESTS="termux-api"`.
